DevOps/frankfurter/scripts/tools/sendlogs.sh

```diff
@@ -0,0 +1,32 @@
+#!/bin/bash
+
+HOME=ubuntu
+USER=192.168.7.2
+
+#format: ssh ubuntu 192.168.7.2 "Linux commands"
+#makes a temp directory and goes into the temp directory once made
+#gets the logs from the services in the beaglebone and logs in temp directory
+#zips log files and prints the zip file name if it has been created
+#exit ssh
+
+ssh $HOME@$USER "
+mkdir -p /tmp/log-fetch
+cd /temp/log-fetch
+
+journalctl -e -u update.service > update.log
+journalctl -e -u runtime.service > service.log
+journalctl -e -u networking.service > networking.log 
+
```

Comment:
try adding a time-stamp to the name of each log so that it is easier to recognize